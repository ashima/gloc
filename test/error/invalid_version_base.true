File 0, line 1, col 9 - 13:
version must be specified in decimal

