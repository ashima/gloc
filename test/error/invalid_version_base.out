#version 0144/*  can only be decimal '100' no more no less and always*/
